nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-5-processors/src/test/java/org/apache/nifi/processors/elasticsearch/TestPutElasticsearch5.java

```diff
@@ -250,6 +250,16 @@ public void testPutElasticsearch5OnTriggerWithExceptions() throws IOException {
 
         // This test generates an exception on execute(),routes to failure
         runner.assertTransferCount(PutElasticsearch5.REL_FAILURE, 1);
+        
+     // Node Closed exception
+        processor.setExceptionToThrow(new NodeClosedException(mock(StreamInput.class)));
```

Comment:
Is this test related to the version stuff? As written it is causing the test to fail, as the exception will cause the flow file to be transferred to FAILURE not RETRY.